components/Tabs/tests/unit/Theming/TabsThemingTests.m

```diff
@@ -0,0 +1,191 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import <MaterialComponents/MaterialColorScheme.h>
```

Comment:
These framework-style imports shouldn't be necessary since this code and the schemes are both in the Beta podspec.

Reply:
I don't think ColorScheme is in Beta. Did you leave the comment at the wrong place?

Reply:
Yep, sorry. Mixed-up in my head with Container.